templates/partnerships/partners/partners_list.html

```diff
@@ -53,51 +55,51 @@ <h1>{% trans 'partners' %}</h1>
 {% url 'organization_edit' organization_id=organization.id as url_identification_edit %}
 {% has_perm 'partnership.add_partner' user as can_add_partner %}
 
-<div class="panel panel-default">
-  <div class="panel-body">
-
-    <div class="well">
+<div class="card">
+  <div class="card-body">
+    <div class="card-header border p-3">
       <div class="row">
-        <div class="col-sm-12">
-          <div class="pull-right">
+        <div class="col-12">
+          <div class="float-end">
             {% if can_add_partner %}
-            <a href="{% url "partnerships:partners:create" %}" class="btn btn-default"><i class="fas fa-fw fa-plus"></i> {% trans "add_partner" %}</a>
+            <a href="{% url 'partnerships:partners:create' %}" class="btn bg-white border border-dark-subtle"><i class="fas fa-fw fa-plus"></i> {% trans "add_partner" %}</a>
             {% endif %}
-            <button class="btn btn-default" type="button"
-                    data-toggle="collapse" data-target="#partner-filters"
+            <button class="btn bg-white border border-dark-subtle"
+                    type="button"
+                    data-bs-toggle="collapse" data-bs-target="#partner-filters"
                     aria-expanded="false" aria-controls="partner-filters">
-              <i class="fas fa-filter"></i> <span class="hidden-xs">{% trans "search_partners" %}</span>
+              <i class="fas fa-filter"></i> <span class="d-none d-md-inline">{% trans "search_partners" %}</span>
             </button>
           </div>
         </div>
       </div>
       {% with form=filter.form %}
-        <div class="partner-form collapse{% if form.is_bound %} in{% endif %}" id="partner-filters">
-          <form action="{% url "partnerships:partners:list" %}" id="filter-form" method="get" autocomplete="off">
+        <div class="partner-form collapse{% if form.is_bound %} show{% endif %}" id="partner-filters">
+          <form action="{% url 'partnerships:partners:list' %}" id="filter-form" method="get" autocomplete="off">
             <div class="row">
-              <div class="col-sm-12">{% bootstrap_field form.name %}</div>
+              <div class="col-12">{% bootstrap_field form.name %}</div>
             </div>
             <div class="row">
-              <div class="col-sm-12 col-md-4">{% bootstrap_field form.partner_type %}</div>
-              <div class="col-sm-12 col-md-4">{% bootstrap_field form.pic_code %}</div>
-              <div class="col-sm-12 col-md-4">{% bootstrap_field form.erasmus_code %}</div>
+              <div class="col-12 col-lg-4">{% bootstrap_field form.partner_type bound_css_class=''  %}</div>
```

Comment:
Mmm I don't know this, what is it ?

Reply:
as i have made configuration for django_BS5 it is useless now i can remove it now, it is use to give a class to the input